src/components/data-exploration/differential-expression-tool/DiffExprResults.jsx

```diff
@@ -36,7 +36,6 @@ const DiffExprResults = (props) => {
 
   const columns = [
     {
-      fixed: 'left',
```

Comment:
Can you explain why you do these modifications on the fixed property and on `GeneTable` too?

Reply:
`fixed: 'left'` fixes the column with this property to the left of the table if the table's width is shorter than its container's width. I removed the property from `logFC` because it is not the left most column. The left most column is actually gene name and the column with with checkboxes. That's why added them there.

Before : 
![image](https://user-images.githubusercontent.com/2862362/143463466-d001712e-e42d-4f39-bc4c-4e935a58606f.png)

After : 
![image](https://user-images.githubusercontent.com/2862362/143463496-51dd0ff0-19ab-4981-a4ea-a8b643e27b49.png)